[Bug]: `panic: value is null` on decoding `ReadResource` result for `aws_lb_listener_rule` in version `v5.63.1`

### Terraform Core Version

1.8.5

### AWS Provider Version

5.63.1

### Affected Resource(s)

`aws_lb_listener_rule`

### Expected Behavior

While trying to extract the existing state of an `aws_lb_listener_rule` resource. the `terraform-provider-aws` should to decode the response correctly.

### Actual Behavior

While trying to extract the existing state of an `aws_lb_listener_rule` resource. the `terraform-provider-aws` fails to decode the response  and throws an error `panic: value is null`.

### Relevant Error/Panic Output Snippet

_No response_

### Terraform Configuration Files

```tf
terraform {
  required_providers {
    aws = {
      source = "hashicorp/aws"
      version = "~> 5.63.1"
    }
  }
}

provider "aws" {
  region = "ap-south-1"
  profile="stage"
}```

### Steps to Reproduce

- Create the above provider configuration
- Create aws credentials profile `stage`
- `tf init`
- download `terrformer`
- run `terraformer import aws -v --resources=alb --regions=ap-south-1 --profile=stage` against any AWS account that has any load balancer configuration with listener rules and target groups.

### Debug Output

```log
2024-08-25T03:25:48.633+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: [DEBUG] Waiting for state to become: [success]
2024-08-25T03:25:48.633+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: HTTP Request Sent: @module=aws http.method=POST rpc.method=DescribeRules rpc.service="Elastic Load Balancing v2" tf_resource_type=aws_lb_listener_rule http.user_agent="APN/1.0 HashiCorp/1.0 Terraform/0.12.31 (+https://www.terraform.io) terraform-provider-aws/5.63.1 (+https://registry.terraform.io/providers/hashicorp/aws) aws-sdk-go-v2/1.30.4 os/linux lang/go#1.22.6 md/GOOS#linux md/GOARCH#amd64 api/elasticloadbalancingv2#1.34.1" rpc.system=aws-api tf_rpc=ReadResource tf_req_id=2358e720-97b5-0a90-a325-4ada6c6e24ad @caller=github.com/hashicorp/aws-sdk-go-base/v2@v2.0.0-beta.54/logging/tf_logger.go:45 http.request.header.authorization="AWS4-HMAC-SHA256 Credential=AKIA************2ZG3/20240824/ap-south-1/elasticloadbalancing/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-request;content-length;content-type;host;x-amz-date, Signature=*****" http.request.header.x_amz_date=20240824T215548Z http.url=https://elasticloadbalancing.ap-south-1.amazonaws.com/ tf_aws.sdk=aws-sdk-go-v2 aws.region=ap-south-1 http.request_content_length=210
  http.request.body=
  | Action=DescribeRules&RuleArns.member.1=arn%3Aaws%3Aelasticloadbalancing%3Aap-south-1%3A813659156632%3Alistener-rule%2Fapp%2Fstaging-lb%2F4fe491221f35cc6e%2F04ee9972066da60f%2F6ce4581d18e338ca&Version=2015-12-01
   http.request.header.content_type=application/x-www-form-urlencoded tf_provider_addr=registry.terraform.io/hashicorp/aws http.request.header.amz_sdk_invocation_id=5de1ba87-e390-49b8-a68d-dedc190906a1 http.request.header.amz_sdk_request="attempt=1; max=25" net.peer.name=elasticloadbalancing.ap-south-1.amazonaws.com tf_aws.signing_region="" tf_mux_provider="*schema.GRPCProviderServer" timestamp="2024-08-25T03:25:48.628+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Received request: @module=sdk.proto tf_resource_type=aws_lb_target_group_attachment tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:771 tf_proto_version=5.6 tf_provider_addr=registry.terraform.io/hashicorp/aws tf_req_id=d6aca2db-e9ea-3e54-6a33-4883cc743fd4 timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: No announced client capabilities: tf_proto_version=5.6 tf_provider_addr=registry.terraform.io/hashicorp/aws tf_req_id=d6aca2db-e9ea-3e54-6a33-4883cc743fd4 @module=sdk.proto tf_resource_type=aws_lb_target_group_attachment tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/client_capabilities.go:44 timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Sending request downstream: tf_proto_version=5.6 tf_req_id=d6aca2db-e9ea-3e54-6a33-4883cc743fd4 tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/downstream_request.go:22 @module=sdk.proto tf_provider_addr=registry.terraform.io/hashicorp/aws tf_resource_type=aws_lb_target_group_attachment timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: calling downstream server: tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-mux@v0.16.0/internal/logging/mux.go:19 @module=sdk.mux tf_mux_provider="*schema.GRPCProviderServer" timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: HTTP Request Sent: http.request.header.amz_sdk_invocation_id=8ac2495e-1c0c-421b-99e0-324053ac93db http.request.header.x_amz_date=20240824T215548Z tf_req_id=73bfc2ba-67e1-72d0-8780-e965756b9fe7
  http.request.body=
  | Action=DescribeRules&RuleArns.member.1=arn%3Aaws%3Aelasticloadbalancing%3Aap-south-1%3A813659156632%3Alistener-rule%2Fapp%2Fstaging-lb%2F4fe491221f35cc6e%2F04ee9972066da60f%2Fdb38e22ef6033dfc&Version=2015-12-01
   rpc.method=DescribeRules http.request.header.authorization="AWS4-HMAC-SHA256 Credential=AKIA************2ZG3/20240824/ap-south-1/elasticloadbalancing/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-request;content-length;content-type;host;x-amz-date, Signature=*****" http.request_content_length=210 http.url=https://elasticloadbalancing.ap-south-1.amazonaws.com/ tf_aws.signing_region="" tf_provider_addr=registry.terraform.io/hashicorp/aws @caller=github.com/hashicorp/aws-sdk-go-base/v2@v2.0.0-beta.54/logging/tf_logger.go:45 http.request.header.amz_sdk_request="attempt=1; max=25" http.request.header.content_type=application/x-www-form-urlencoded tf_resource_type=aws_lb_listener_rule tf_rpc=ReadResource rpc.system=aws-api http.method=POST net.peer.name=elasticloadbalancing.ap-south-1.amazonaws.com tf_mux_provider="*schema.GRPCProviderServer" aws.region=ap-south-1 rpc.service="Elastic Load Balancing v2" @module=aws http.user_agent="APN/1.0 HashiCorp/1.0 Terraform/0.12.31 (+https://www.terraform.io) terraform-provider-aws/5.63.1 (+https://registry.terraform.io/providers/hashicorp/aws) aws-sdk-go-v2/1.30.4 os/linux lang/go#1.22.6 md/GOOS#linux md/GOARCH#amd64 api/elasticloadbalancingv2#1.34.1" tf_aws.sdk=aws-sdk-go-v2 timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Received request: tf_resource_type=aws_lb_target_group tf_rpc=ReadResource tf_proto_version=5.6 tf_req_id=93f18840-63fb-0be5-622f-c206c397d123 tf_provider_addr=registry.terraform.io/hashicorp/aws @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:771 @module=sdk.proto timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Calling downstream: @module=sdk.helper_schema tf_req_id=d6aca2db-e9ea-3e54-6a33-4883cc743fd4 tf_mux_provider="*schema.GRPCProviderServer" tf_provider_addr=registry.terraform.io/hashicorp/aws tf_resource_type=aws_lb_target_group_attachment tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:1116 timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Received request: @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:771 @module=sdk.proto tf_proto_version=5.6 tf_req_id=5c91ca1a-0c64-e0ce-5387-62e5968ed6f2 tf_provider_addr=registry.terraform.io/hashicorp/aws tf_resource_type=aws_lb_target_group_attachment tf_rpc=ReadResource timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Received request: @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:771 tf_proto_version=5.6 tf_req_id=3c2b7975-27bd-7792-a79f-0dc75aaac131 @module=sdk.proto tf_provider_addr=registry.terraform.io/hashicorp/aws tf_resource_type=aws_lb_listener_rule tf_rpc=ReadResource timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: No announced client capabilities: @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/client_capabilities.go:44 @module=sdk.proto tf_proto_version=5.6 tf_provider_addr=registry.terraform.io/hashicorp/aws tf_resource_type=aws_lb_target_group_attachment tf_rpc=ReadResource tf_req_id=5c91ca1a-0c64-e0ce-5387-62e5968ed6f2 timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: No announced client capabilities: @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/client_capabilities.go:44 @module=sdk.proto tf_provider_addr=registry.terraform.io/hashicorp/aws tf_req_id=3c2b7975-27bd-7792-a79f-0dc75aaac131 tf_resource_type=aws_lb_listener_rule tf_proto_version=5.6 tf_rpc=ReadResource timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Sending request downstream: @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/downstream_request.go:22 tf_provider_addr=registry.terraform.io/hashicorp/aws @module=sdk.proto tf_proto_version=5.6 tf_req_id=5c91ca1a-0c64-e0ce-5387-62e5968ed6f2 tf_resource_type=aws_lb_target_group_attachment tf_rpc=ReadResource timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: No announced client capabilities: @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/client_capabilities.go:44 tf_req_id=93f18840-63fb-0be5-622f-c206c397d123 tf_resource_type=aws_lb_target_group @module=sdk.proto tf_proto_version=5.6 tf_provider_addr=registry.terraform.io/hashicorp/aws tf_rpc=ReadResource timestamp="2024-08-25T03:25:48.629+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Sending request downstream: tf_rpc=ReadResource tf_provider_addr=registry.terraform.io/hashicorp/aws tf_req_id=3c2b7975-27bd-7792-a79f-0dc75aaac131 tf_resource_type=aws_lb_listener_rule tf_proto_version=5.6 @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/downstream_request.go:22 @module=sdk.proto timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Sending request downstream: @module=sdk.proto tf_proto_version=5.6 tf_provider_addr=registry.terraform.io/hashicorp/aws tf_req_id=93f18840-63fb-0be5-622f-c206c397d123 tf_resource_type=aws_lb_target_group tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/downstream_request.go:22 timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: calling downstream server: @caller=github.com/hashicorp/terraform-plugin-mux@v0.16.0/internal/logging/mux.go:19 @module=sdk.mux tf_mux_provider="*schema.GRPCProviderServer" tf_rpc=ReadResource timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: calling downstream server: @module=sdk.mux tf_mux_provider="*schema.GRPCProviderServer" tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-mux@v0.16.0/internal/logging/mux.go:19 timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: calling downstream server: tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-mux@v0.16.0/internal/logging/mux.go:19 @module=sdk.mux tf_mux_provider="*schema.GRPCProviderServer" timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Received request: @module=sdk.proto tf_provider_addr=registry.terraform.io/hashicorp/aws tf_rpc=ReadResource tf_req_id=711d06f0-fa6c-4902-8167-0609c8595e88 tf_resource_type=aws_lb_target_group @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:771 tf_proto_version=5.6 timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: No announced client capabilities: tf_resource_type=aws_lb_target_group tf_proto_version=5.6 tf_provider_addr=registry.terraform.io/hashicorp/aws @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/client_capabilities.go:44 @module=sdk.proto tf_req_id=711d06f0-fa6c-4902-8167-0609c8595e88 tf_rpc=ReadResource timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.633+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Calling downstream: tf_mux_provider="*schema.GRPCProviderServer" tf_provider_addr=registry.terraform.io/hashicorp/aws tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:1116 @module=sdk.helper_schema tf_resource_type=aws_lb_target_group_attachment tf_req_id=5c91ca1a-0c64-e0ce-5387-62e5968ed6f2 timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.634+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Sending request downstream: @module=sdk.proto tf_proto_version=5.6 tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/downstream_request.go:22 tf_provider_addr=registry.terraform.io/hashicorp/aws tf_req_id=711d06f0-fa6c-4902-8167-0609c8595e88 tf_resource_type=aws_lb_target_group timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.634+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Calling downstream: @caller=github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:1116 @module=sdk.helper_schema tf_provider_addr=registry.terraform.io/hashicorp/aws tf_resource_type=aws_lb_listener_rule tf_rpc=ReadResource tf_mux_provider="*schema.GRPCProviderServer" tf_req_id=3c2b7975-27bd-7792-a79f-0dc75aaac131 timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.634+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: [DEBUG] Waiting for state to become: [success]
2024-08-25T03:25:48.634+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: calling downstream server: @module=sdk.mux tf_mux_provider="*schema.GRPCProviderServer" tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-mux@v0.16.0/internal/logging/mux.go:19 timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.634+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Calling downstream: tf_resource_type=aws_lb_target_group tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:1116 @module=sdk.helper_schema tf_mux_provider="*schema.GRPCProviderServer" tf_provider_addr=registry.terraform.io/hashicorp/aws tf_req_id=93f18840-63fb-0be5-622f-c206c397d123 timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.634+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: HTTP Request Sent: rpc.method=DescribeTargetHealth tf_rpc=ReadResource http.request.header.content_type=application/x-www-form-urlencoded net.peer.name=elasticloadbalancing.ap-south-1.amazonaws.com tf_mux_provider="*schema.GRPCProviderServer" tf_req_id=d6aca2db-e9ea-3e54-6a33-4883cc743fd4 tf_resource_type=aws_lb_target_group_attachment http.request.header.x_amz_date=20240824T215548Z rpc.service="Elastic Load Balancing v2" tf_aws.sdk=aws-sdk-go-v2 @caller=github.com/hashicorp/aws-sdk-go-base/v2@v2.0.0-beta.54/logging/tf_logger.go:45 http.request.header.amz_sdk_invocation_id=bc2d067e-f913-4bce-bdd7-c7df7839e82a @module=aws http.request.header.authorization="AWS4-HMAC-SHA256 Credential=AKIA************2ZG3/20240824/ap-south-1/elasticloadbalancing/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-request;content-length;content-type;host;x-amz-date, Signature=*****" rpc.system=aws-api tf_aws.signing_region=""
  http.request.body=
  | Action=DescribeTargetHealth&TargetGroupArn=arn%3Aaws%3Aelasticloadbalancing%3Aap-south-1%3A813659156632%3Atargetgroup%2Fstage-1-TG%2F51337aac92427263&Targets.member.1.Id=10.3.2.143&Version=2015-12-01
   http.request_content_length=204 http.url=https://elasticloadbalancing.ap-south-1.amazonaws.com/ tf_provider_addr=registry.terraform.io/hashicorp/aws aws.region=ap-south-1 http.method=POST http.request.header.amz_sdk_request="attempt=1; max=25" http.user_agent="APN/1.0 HashiCorp/1.0 Terraform/0.12.31 (+https://www.terraform.io) terraform-provider-aws/5.63.1 (+https://registry.terraform.io/providers/hashicorp/aws) aws-sdk-go-v2/1.30.4 os/linux lang/go#1.22.6 md/GOOS#linux md/GOARCH#amd64 api/elasticloadbalancingv2#1.34.1" timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.634+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Received request: tf_resource_type=aws_lb_listener_rule tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:771 @module=sdk.proto tf_proto_version=5.6 tf_provider_addr=registry.terraform.io/hashicorp/aws tf_req_id=2d418485-c228-dd3c-91f3-93f77c47e383 timestamp="2024-08-25T03:25:48.630+0530"
2024-08-25T03:25:48.634+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: No announced client capabilities: @module=sdk.proto tf_proto_version=5.6 tf_provider_addr=registry.terraform.io/hashicorp/aws tf_req_id=2d418485-c228-dd3c-91f3-93f77c47e383 tf_resource_type=aws_lb_listener_rule @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/client_capabilities.go:44 tf_rpc=ReadResource timestamp="2024-08-25T03:25:48.631+0530"
2024-08-25T03:25:48.634+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Sending request downstream: tf_req_id=2d418485-c228-dd3c-91f3-93f77c47e383 tf_resource_type=aws_lb_listener_rule @caller=github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tf5serverlogging/downstream_request.go:22 tf_provider_addr=registry.terraform.io/hashicorp/aws tf_rpc=ReadResource @module=sdk.proto tf_proto_version=5.6 timestamp="2024-08-25T03:25:48.631+0530"
2024-08-25T03:25:48.634+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: calling downstream server: @module=sdk.mux tf_mux_provider="*schema.GRPCProviderServer" tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-mux@v0.16.0/internal/logging/mux.go:19 timestamp="2024-08-25T03:25:48.631+0530"
2024-08-25T03:25:48.634+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Calling downstream: tf_req_id=711d06f0-fa6c-4902-8167-0609c8595e88 tf_resource_type=aws_lb_target_group tf_rpc=ReadResource @module=sdk.helper_schema tf_provider_addr=registry.terraform.io/hashicorp/aws @caller=github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:1116 tf_mux_provider="*schema.GRPCProviderServer" timestamp="2024-08-25T03:25:48.631+0530"
2024-08-25T03:25:48.634+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Calling downstream: tf_mux_provider="*schema.GRPCProviderServer" tf_req_id=2d418485-c228-dd3c-91f3-93f77c47e383 tf_provider_addr=registry.terraform.io/hashicorp/aws tf_resource_type=aws_lb_listener_rule tf_rpc=ReadResource @caller=github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:1116 @module=sdk.helper_schema timestamp="2024-08-25T03:25:48.631+0530"
2024-08-25T03:25:48.634+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: HTTP Request Sent: http.request.header.amz_sdk_invocation_id=7f64b503-0836-4875-b6be-e9029a51fdb7 http.request.header.amz_sdk_request="attempt=1; max=25" http.request.header.x_amz_date=20240824T215548Z tf_aws.signing_region="" http.request.header.authorization="AWS4-HMAC-SHA256 Credential=AKIA************2ZG3/20240824/ap-south-1/elasticloadbalancing/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-request;content-length;content-type;host;x-amz-date, Signature=*****" http.request.header.content_type=application/x-www-form-urlencoded http.method=POST net.peer.name=elasticloadbalancing.ap-south-1.amazonaws.com rpc.method=DescribeRules rpc.service="Elastic Load Balancing v2" tf_rpc=ReadResource @caller=github.com/hashicorp/aws-sdk-go-base/v2@v2.0.0-beta.54/logging/tf_logger.go:45 http.request_content_length=210 http.url=https://elasticloadbalancing.ap-south-1.amazonaws.com/ rpc.system=aws-api http.user_agent="APN/1.0 HashiCorp/1.0 Terraform/0.12.31 (+https://www.terraform.io) terraform-provider-aws/5.63.1 (+https://registry.terraform.io/providers/hashicorp/aws) aws-sdk-go-v2/1.30.4 os/linux lang/go#1.22.6 md/GOOS#linux md/GOARCH#amd64 api/elasticloadbalancingv2#1.34.1" tf_aws.sdk=aws-sdk-go-v2 tf_req_id=3c2b7975-27bd-7792-a79f-0dc75aaac131 @module=aws aws.region=ap-south-1
  http.request.body=
  | Action=DescribeRules&RuleArns.member.1=arn%3Aaws%3Aelasticloadbalancing%3Aap-south-1%3A813659156632%3Alistener-rule%2Fapp%2Fstaging-lb%2F4fe491221f35cc6e%2F04ee9972066da60f%2Fd0c8a2af379bf28f&Version=2015-12-01
   tf_mux_provider="*schema.GRPCProviderServer" tf_provider_addr=registry.terraform.io/hashicorp/aws tf_resource_type=aws_lb_listener_rule timestamp="2024-08-25T03:25:48.631+0530"
2024-08-25T03:25:48.634+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: [DEBUG] Waiting for state to become: [success]
2024-08-25T03:25:48.634+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: HTTP Request Sent: http.user_agent="APN/1.0 HashiCorp/1.0 Terraform/0.12.31 (+https://www.terraform.io) terraform-provider-aws/5.63.1 (+https://registry.terraform.io/providers/hashicorp/aws) m/C aws-sdk-go-v2/1.30.4 os/linux lang/go#1.22.6 md/GOOS#linux md/GOARCH#amd64 api/elasticloadbalancingv2#1.34.1" tf_aws.sdk=aws-sdk-go-v2 http.request.header.amz_sdk_request="attempt=1; max=25" http.request_content_length=178 rpc.system=aws-api tf_resource_type=aws_lb_target_group @caller=github.com/hashicorp/aws-sdk-go-base/v2@v2.0.0-beta.54/logging/tf_logger.go:45 aws.region=ap-south-1 http.method=POST http.request.header.amz_sdk_invocation_id=f1e0c303-68c8-4558-9409-028aa7fe7698 tf_req_id=93f18840-63fb-0be5-622f-c206c397d123 http.url=https://elasticloadbalancing.ap-south-1.amazonaws.com/ net.peer.name=elasticloadbalancing.ap-south-1.amazonaws.com tf_rpc=ReadResource @module=aws http.request.header.content_type=application/x-www-form-urlencoded http.request.header.x_amz_date=20240824T215548Z rpc.service="Elastic Load Balancing v2"
  http.request.body=
  | Action=DescribeTargetGroups&TargetGroupArns.member.1=arn%3Aaws%3Aelasticloadbalancing%3Aap-south-1%3A813659156632%3Atargetgroup%2Fstage-2-TG%2F8a7ca35e9ee48e40&Version=2015-12-01
   http.request.header.authorization="AWS4-HMAC-SHA256 Credential=AKIA************2ZG3/20240824/ap-south-1/elasticloadbalancing/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-request;content-length;content-type;host;x-amz-date, Signature=*****" rpc.method=DescribeTargetGroups tf_aws.signing_region="" tf_mux_provider="*schema.GRPCProviderServer" tf_provider_addr=registry.terraform.io/hashicorp/aws timestamp="2024-08-25T03:25:48.631+0530"
2024-08-25T03:25:48.634+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: HTTP Request Sent: aws.region=ap-south-1 http.request.header.authorization="AWS4-HMAC-SHA256 Credential=AKIA************2ZG3/20240824/ap-south-1/elasticloadbalancing/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-request;content-length;content-type;host;x-amz-date, Signature=*****" http.request.header.content_type=application/x-www-form-urlencoded http.request_content_length=298 http.request.header.amz_sdk_invocation_id=db2d69db-6827-4381-93a6-c3bc763a525c http.request.header.x_amz_date=20240824T215548Z http.url=https://elasticloadbalancing.ap-south-1.amazonaws.com/ rpc.system=aws-api @caller=github.com/hashicorp/aws-sdk-go-base/v2@v2.0.0-beta.54/logging/tf_logger.go:45 @module=aws tf_aws.signing_region="" tf_provider_addr=registry.terraform.io/hashicorp/aws tf_resource_type=aws_lb_target_group_attachment http.method=POST
  http.request.body=
  | Action=DescribeTargetHealth&TargetGroupArn=arn%3Aaws%3Aelasticloadbalancing%3Aap-south-1%3A813659156632%3Atargetgroup%2Fnlb-TG%2Fc7af9202aab5e9b2&Targets.member.1.Id=arn%3Aaws%3Aelasticloadbalancing%3Aap-south-1%3A813659156632%3Aloadbalancer%2Fapp%2Fstaging-lb%2F4fe491221f35cc6e&Version=2015-12-01
   net.peer.name=elasticloadbalancing.ap-south-1.amazonaws.com rpc.method=DescribeTargetHealth tf_mux_provider="*schema.GRPCProviderServer" http.request.header.amz_sdk_request="attempt=1; max=25" http.user_agent="APN/1.0 HashiCorp/1.0 Terraform/0.12.31 (+https://www.terraform.io) terraform-provider-aws/5.63.1 (+https://registry.terraform.io/providers/hashicorp/aws) aws-sdk-go-v2/1.30.4 os/linux lang/go#1.22.6 md/GOOS#linux md/GOARCH#amd64 api/elasticloadbalancingv2#1.34.1" tf_aws.sdk=aws-sdk-go-v2 rpc.service="Elastic Load Balancing v2" tf_req_id=5c91ca1a-0c64-e0ce-5387-62e5968ed6f2 tf_rpc=ReadResource timestamp="2024-08-25T03:25:48.631+0530"
2024-08-25T03:25:48.634+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: HTTP Request Sent: rpc.service="Elastic Load Balancing v2" rpc.system=aws-api tf_rpc=ReadResource http.user_agent="APN/1.0 HashiCorp/1.0 Terraform/0.12.31 (+https://www.terraform.io) terraform-provider-aws/5.63.1 (+https://registry.terraform.io/providers/hashicorp/aws) m/C aws-sdk-go-v2/1.30.4 os/linux lang/go#1.22.6 md/GOOS#linux md/GOARCH#amd64 api/elasticloadbalancingv2#1.34.1" rpc.method=DescribeTargetGroups tf_aws.signing_region="" tf_resource_type=aws_lb_target_group tf_aws.sdk=aws-sdk-go-v2 tf_req_id=711d06f0-fa6c-4902-8167-0609c8595e88 @caller=github.com/hashicorp/aws-sdk-go-base/v2@v2.0.0-beta.54/logging/tf_logger.go:45
  http.request.body=
  | Action=DescribeTargetGroups&TargetGroupArns.member.1=arn%3Aaws%3Aelasticloadbalancing%3Aap-south-1%3A813659156632%3Atargetgroup%2Fuat-merchant-payout-tg%2Ffe5758048917117f&Version=2015-12-01
   http.request.header.amz_sdk_request="attempt=1; max=25" @module=aws aws.region=ap-south-1 http.request.header.amz_sdk_invocation_id=c16ee51f-9f4c-499a-a5a6-8ba7e163eeb7 net.peer.name=elasticloadbalancing.ap-south-1.amazonaws.com tf_provider_addr=registry.terraform.io/hashicorp/aws http.method=POST http.request.header.content_type=application/x-www-form-urlencoded http.request_content_length=190 tf_mux_provider="*schema.GRPCProviderServer" http.request.header.authorization="AWS4-HMAC-SHA256 Credential=AKIA************2ZG3/20240824/ap-south-1/elasticloadbalancing/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-request;content-length;content-type;host;x-amz-date, Signature=*****" http.request.header.x_amz_date=20240824T215548Z http.url=https://elasticloadbalancing.ap-south-1.amazonaws.com/ timestamp="2024-08-25T03:25:48.632+0530"
2024-08-25T03:25:48.634+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: HTTP Request Sent: http.request.header.amz_sdk_request="attempt=1; max=25" tf_aws.sdk=aws-sdk-go-v2 tf_aws.signing_region="" http.method=POST @module=aws http.request.header.authorization="AWS4-HMAC-SHA256 Credential=AKIA************2ZG3/20240824/ap-south-1/elasticloadbalancing/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-request;content-length;content-type;host;x-amz-date, Signature=*****" rpc.service="Elastic Load Balancing v2" tf_provider_addr=registry.terraform.io/hashicorp/aws tf_resource_type=aws_lb_listener_rule net.peer.name=elasticloadbalancing.ap-south-1.amazonaws.com rpc.system=aws-api tf_rpc=ReadResource http.request_content_length=210 rpc.method=DescribeRules http.request.header.amz_sdk_invocation_id=863c3244-7b2f-4c7f-8c10-26f1bc6a27fa tf_mux_provider="*schema.GRPCProviderServer" tf_req_id=2d418485-c228-dd3c-91f3-93f77c47e383 aws.region=ap-south-1 http.request.header.x_amz_date=20240824T215548Z http.url=https://elasticloadbalancing.ap-south-1.amazonaws.com/ @caller=github.com/hashicorp/aws-sdk-go-base/v2@v2.0.0-beta.54/logging/tf_logger.go:45
  http.request.body=
  | Action=DescribeRules&RuleArns.member.1=arn%3Aaws%3Aelasticloadbalancing%3Aap-south-1%3A813659156632%3Alistener-rule%2Fapp%2Fstaging-lb%2F4fe491221f35cc6e%2F04ee9972066da60f%2Ff4726a461fa0dcd5&Version=2015-12-01
   http.user_agent="APN/1.0 HashiCorp/1.0 Terraform/0.12.31 (+https://www.terraform.io) terraform-provider-aws/5.63.1 (+https://registry.terraform.io/providers/hashicorp/aws) aws-sdk-go-v2/1.30.4 os/linux lang/go#1.22.6 md/GOOS#linux md/GOARCH#amd64 api/elasticloadbalancingv2#1.34.1" http.request.header.content_type=application/x-www-form-urlencoded timestamp="2024-08-25T03:25:48.632+0530"
2024-08-25T03:25:48.791+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: HTTP Response Received: tf_mux_provider="*schema.GRPCProviderServer" @module=aws aws.region=ap-south-1 http.response.header.content_type=text/xml http.response_content_length=1784 tf_aws.signing_region=""
  http.response.body=
  | <DescribeRulesResponse xmlns="http://elasticloadbalancing.amazonaws.com/doc/2015-12-01/">
  |   <DescribeRulesResult>
  |     <Rules>
  |       <member>
  |         <IsDefault>false</IsDefault>
  |         <Conditions>
  |           <member>
  |             <HostHeaderConfig>
  |               <Values>
  |                 <member>my.demo.in</member>
  |               </Values>
  |             </HostHeaderConfig>
  |             <Field>host-header</Field>
  |             <Values>
  |               <member>my.demo.in</member>
  |             </Values>
  |           </member>
  |         </Conditions>
  |         <Priority>86</Priority>
  |         <Actions>
  |           <member>
  |             <TargetGroupArn>arn:aws:elasticloadbalancing:ap-south-1:813659156632:targetgroup/stage-3-TG/887c532f17429f70</TargetGroupArn>
  |             <ForwardConfig>
  |               <TargetGroups>
  |                 <member>
  |                   <TargetGroupArn>arn:aws:elasticloadbalancing:ap-south-1:813659156632:targetgroup/stage-3-TG/887c532f17429f70</TargetGroupArn>
  |                   <Weight>1</Weight>
  |                 </member>
  |               </TargetGroups>
  |               <TargetGroupStickinessConfig>
  |                 <DurationSeconds>3600</DurationSeconds>
  |                 <Enabled>false</Enabled>
  |               </TargetGroupStickinessConfig>
  |             </ForwardConfig>
  |             <Type>forward</Type>
  |             <Order>1</Order>
  |           </member>
  |         </Actions>
  |         <RuleArn>arn:aws:elasticloadbalancing:ap-south-1:813659156632:listener-rule/app/staging-lb/4fe491221f35cc6e/04ee9972066da60f/6ce4581d18e338ca</RuleArn>
  |       </member>
  |     </Rules>
  |   </DescribeRulesResult>
  |   <ResponseMetadata>
  |     <RequestId>2383353e-01d1-4321-b4db-46cb0ac06af9</RequestId>
  |   </ResponseMetadata>
  | </DescribeRulesResponse>
   tf_aws.sdk=aws-sdk-go-v2 tf_rpc=ReadResource tf_resource_type=aws_lb_listener_rule http.response.header.date="Sat, 24 Aug 2024 21:55:48 GMT" http.response.header.x_amzn_requestid=2383353e-01d1-4321-b4db-46cb0ac06af9 http.status_code=200 rpc.method=DescribeRules rpc.system=aws-api tf_provider_addr=registry.terraform.io/hashicorp/aws @caller=github.com/hashicorp/aws-sdk-go-base/v2@v2.0.0-beta.54/logging/tf_logger.go:45 http.duration=161 rpc.service="Elastic Load Balancing v2" tf_req_id=2358e720-97b5-0a90-a325-4ada6c6e24ad timestamp="2024-08-25T03:25:48.790+0530"
2024-08-25T03:25:48.791+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Served request: tf_resource_type=aws_lb_listener_rule tf_rpc=ReadResource @caller=runtime/panic.go:770 @module=sdk.proto tf_req_id=2358e720-97b5-0a90-a325-4ada6c6e24ad tf_proto_version=5.6 tf_provider_addr=registry.terraform.io/hashicorp/aws timestamp="2024-08-25T03:25:48.791+0530"
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: panic: value is null
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: goroutine 28 [running]:
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/go-cty/cty.Value.LengthInt({{{0x17d736f8?, 0xc0031eec60?}}, {0x0?, 0x0?}})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/go-cty@v1.4.1-0.20200723130312-85980079f637/cty/value_ops.go:989 +0x258
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-provider-aws/internal/service/elbv2.flattenLbForwardAction(0xc0022b0180, {0x159ab36d, 0x6}, 0x0, {{0xc000b4e980, 0x7}, 0x0, 0x0, 0x0, 0xc004c96f00, ...}, ...)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-provider-aws/internal/service/elbv2/listener.go:1023 +0x1f8
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-provider-aws/internal/service/elbv2.flattenLbListenerActions(0xc0022b0180, {0x159ab36d, 0x6}, {0xc0022245f0?, 0x1, 0x15d7a648?})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-provider-aws/internal/service/elbv2/listener.go:982 +0x2c5
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-provider-aws/internal/service/elbv2.resourceListenerRuleRead({0x17d733a8, 0xc0047a3320}, 0xc0022b0180, {0x15987360?, 0xc004953520?})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-provider-aws/internal/service/elbv2/listener_rule.go:584 +0x6a5
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-provider-aws/internal/provider.New.(*wrappedResource).Read.interceptedHandler[...].func9(0xc0022b0180?, {0x15987360?, 0xc004953520})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-provider-aws/internal/provider/intercept.go:113 +0x283
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0x17d733a8?, {0x17d733a8?, 0xc002604660?}, 0xd?, {0x15987360?, 0xc004953520?})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:818 +0x7a
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc001612540, {0x17d733a8, 0xc002604660}, 0xc001ac3380, {0x15987360, 0xc004953520})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:1117 +0x529
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc00290fa88, {0x17d733a8?, 0xc0026045a0?}, 0xc002bfa0c0)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/grpc_provider.go:708 +0x6c5
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).ReadResource(0xc000f4a310, {0x17d733a8?, 0xc002604090?}, 0xc002bfa0c0)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-mux@v0.16.0/tf5muxserver/mux_server_ReadResource.go:35 +0x193
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ReadResource(0xc0005f3b80, {0x17d733a8?, 0xc0025e62a0?}, 0xc001ba0000)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:783 +0x309
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler({0x1579aee0, 0xc0005f3b80}, {0x17d733a8, 0xc0025e62a0}, 0xc0035ec480, 0x0)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:482 +0x1a6
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: google.golang.org/grpc.(*Server).processUnaryRPC(0xc000bccc00, {0x17d733a8, 0xc0025e6210}, {0x17ddf980, 0xc0017d7b00}, 0xc0035fb560, 0xc004a08a20, 0x215e6ee8, 0x0)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	google.golang.org/grpc@v1.63.2/server.go:1369 +0xdf8
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: google.golang.org/grpc.(*Server).handleStream(0xc000bccc00, {0x17ddf980, 0xc0017d7b00}, 0xc0035fb560)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	google.golang.org/grpc@v1.63.2/server.go:1780 +0xe8b
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: google.golang.org/grpc.(*Server).serveStreams.func2.1()
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	google.golang.org/grpc@v1.63.2/server.go:1019 +0x8b
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 51
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	google.golang.org/grpc@v1.63.2/server.go:1030 +0x125
2024-08-25T03:25:48.802+0530 [ERROR] plugin: plugin process exited: path=.terraform/providers/registry.terraform.io/hashicorp/aws/5.63.1/linux_amd64/terraform-provider-aws_v5.63.1_x5 pid=96506 error="exit status 2"
```

### Panic Output

```
2024-08-25T03:25:48.791+0530 [TRACE] plugin.terraform-provider-aws_v5.63.1_x5: Served request: tf_resource_type=aws_lb_listener_rule tf_rpc=ReadResource @caller=runtime/panic.go:770 @module=sdk.proto tf_req_id=2358e720-97b5-0a90-a325-4ada6c6e24ad tf_proto_version=5.6 tf_provider_addr=registry.terraform.io/hashicorp/aws timestamp="2024-08-25T03:25:48.791+0530"
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: panic: value is null
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: goroutine 28 [running]:
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/go-cty/cty.Value.LengthInt({{{0x17d736f8?, 0xc0031eec60?}}, {0x0?, 0x0?}})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/go-cty@v1.4.1-0.20200723130312-85980079f637/cty/value_ops.go:989 +0x258
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-provider-aws/internal/service/elbv2.flattenLbForwardAction(0xc0022b0180, {0x159ab36d, 0x6}, 0x0, {{0xc000b4e980, 0x7}, 0x0, 0x0, 0x0, 0xc004c96f00, ...}, ...)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-provider-aws/internal/service/elbv2/listener.go:1023 +0x1f8
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-provider-aws/internal/service/elbv2.flattenLbListenerActions(0xc0022b0180, {0x159ab36d, 0x6}, {0xc0022245f0?, 0x1, 0x15d7a648?})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-provider-aws/internal/service/elbv2/listener.go:982 +0x2c5
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-provider-aws/internal/service/elbv2.resourceListenerRuleRead({0x17d733a8, 0xc0047a3320}, 0xc0022b0180, {0x15987360?, 0xc004953520?})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-provider-aws/internal/service/elbv2/listener_rule.go:584 +0x6a5
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-provider-aws/internal/provider.New.(*wrappedResource).Read.interceptedHandler[...].func9(0xc0022b0180?, {0x15987360?, 0xc004953520})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-provider-aws/internal/provider/intercept.go:113 +0x283
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0x17d733a8?, {0x17d733a8?, 0xc002604660?}, 0xd?, {0x15987360?, 0xc004953520?})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:818 +0x7a
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc001612540, {0x17d733a8, 0xc002604660}, 0xc001ac3380, {0x15987360, 0xc004953520})
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:1117 +0x529
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc00290fa88, {0x17d733a8?, 0xc0026045a0?}, 0xc002bfa0c0)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/grpc_provider.go:708 +0x6c5
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).ReadResource(0xc000f4a310, {0x17d733a8?, 0xc002604090?}, 0xc002bfa0c0)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-mux@v0.16.0/tf5muxserver/mux_server_ReadResource.go:35 +0x193
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ReadResource(0xc0005f3b80, {0x17d733a8?, 0xc0025e62a0?}, 0xc001ba0000)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:783 +0x309
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler({0x1579aee0, 0xc0005f3b80}, {0x17d733a8, 0xc0025e62a0}, 0xc0035ec480, 0x0)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:482 +0x1a6
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: google.golang.org/grpc.(*Server).processUnaryRPC(0xc000bccc00, {0x17d733a8, 0xc0025e6210}, {0x17ddf980, 0xc0017d7b00}, 0xc0035fb560, 0xc004a08a20, 0x215e6ee8, 0x0)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	google.golang.org/grpc@v1.63.2/server.go:1369 +0xdf8
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: google.golang.org/grpc.(*Server).handleStream(0xc000bccc00, {0x17ddf980, 0xc0017d7b00}, 0xc0035fb560)
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	google.golang.org/grpc@v1.63.2/server.go:1780 +0xe8b
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: google.golang.org/grpc.(*Server).serveStreams.func2.1()
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	google.golang.org/grpc@v1.63.2/server.go:1019 +0x8b
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 51
2024-08-25T03:25:48.794+0530 [DEBUG] plugin.terraform-provider-aws_v5.63.1_x5: 	google.golang.org/grpc@v1.63.2/server.go:1030 +0x125
```

### Important Factoids

I am trying to import my existing load balancer configuration into terraform management, for which I am using `terraformer` . However I expect there is some mismatch between the existing state of the load balancer and terraform created ones where the export isn't working.  Tracing the failing request via it's request ID, I see that the request that fails  is for `DescribeRules`  on `RuleArns.member.1` `arn:aws:elasticloadbalancing:ap-south-1:813659156632:listener-rule/app/staging-lb/4fe491221f35cc6e/04ee9972066da60f/6ce4581d18e338ca&Version=2015-12-01`

`$ aws --region 'ap-south-1' --profile=stage elbv2 describe-rules --rule-arns 'arn:aws:elasticloadbalancing:ap-south-1:813659156632:listener-rule/app/staging-lb/4fe491221f35cc6e/04ee9972066da60f/6ce4581d18e338ca'`
```json
{
    "Rules": [
        {
            "RuleArn": "arn:aws:elasticloadbalancing:ap-south-1:813659156632:listener-rule/app/staging-lb/4fe491221f35cc6e/04ee9972066da60f/6ce4581d18e338ca",
            "Priority": "86",
            "Conditions": [
                {
                    "Field": "host-header",
                    "Values": [
                        "my.demo.in"
                    ],
                    "HostHeaderConfig": {
                        "Values": [
                            "my.demo.in"
                        ]
                    }
                }
            ],
            "Actions": [
                {
                    "Type": "forward",
                    "TargetGroupArn": "arn:aws:elasticloadbalancing:ap-south-1:813659156632:targetgroup/stage-3-TG/887c532f17429f70",
                    "Order": 1,
                    "ForwardConfig": {
                        "TargetGroups": [
                            {
                                "TargetGroupArn": "arn:aws:elasticloadbalancing:ap-south-1:813659156632:targetgroup/stage-3-TG/887c532f17429f70",
                                "Weight": 1
                            }
                        ],
                        "TargetGroupStickinessConfig": {
                            "Enabled": false,
                            "DurationSeconds": 3600
                        }
                    }
                }
            ],
            "IsDefault": false
        }
    ]
}
```

The linked target-group:
`$ aws --region 'ap-south-1' --profile=srsstage elbv2 describe-target-groups --target-group-arns 'arn:aws:elasticloadbalancing:ap-south-1:813659156632:targetgroup/stage-3-TG/887c532f17429f70'`

```json
{
    "TargetGroups": [
        {
            "TargetGroupArn": "arn:aws:elasticloadbalancing:ap-south-1:813659156632:targetgroup/stage-3-TG/887c532f17429f70",
            "TargetGroupName": "stage-3-TG",
            "Protocol": "HTTP",
            "Port": 80,
            "VpcId": "vpc-05c1789b2cd022d3b",
            "HealthCheckProtocol": "HTTP",
            "HealthCheckPort": "traffic-port",
            "HealthCheckEnabled": true,
            "HealthCheckIntervalSeconds": 30,
            "HealthCheckTimeoutSeconds": 5,
            "HealthyThresholdCount": 5,
            "UnhealthyThresholdCount": 2,
            "HealthCheckPath": "/",
            "Matcher": {
                "HttpCode": "200"
            },
            "LoadBalancerArns": [
                "arn:aws:elasticloadbalancing:ap-south-1:813659156632:loadbalancer/app/staging-lb/4fe491221f35cc6e"
            ],
            "TargetType": "ip",
            "ProtocolVersion": "HTTP1",
            "IpAddressType": "ipv4"
        }
    ]
}
```

### References

I did not find any relevant issues among the existing ones.

### Would you like to implement a fix?

None

--- a/.changelog/39039.txt
+++ b/.changelog/39039.txt
@@ -1,0 +1,3 @@
+```release-note:bug
+resource/aws_elbv2_listener: Fix crash when reading forward actions not configured in state
+```

--- a/internal/service/elbv2/listener.go
+++ b/internal/service/elbv2/listener.go
@@ -1020,7 +1020,7 @@ func flattenLbForwardAction(d *schema.ResourceData, attrName string, i int, awsA
 	}
 
 	if rawState := d.GetRawState(); rawState.IsKnown() && !rawState.IsNull() {
-		if defaultActions := rawState.GetAttr(attrName); defaultActions.LengthInt() > 0 {
+		if defaultActions := rawState.GetAttr(attrName); defaultActions.IsKnown() && !defaultActions.IsNull() && defaultActions.LengthInt() > 0 {
 			flattenLbForwardActionOneOf(defaultActions, i, awsAction, actionMap)
 			return
 		}

